b2sdk/_internal/raw_api.py

```diff
@@ -76,8 +76,6 @@
     'shareFiles',
     'writeFiles',
     'deleteFiles',
-    'readBucketNotifications',
-    'writeBucketNotifications',
```

Comment:
I re-added these lines and marked `test.integration.test_bucket.test_bucket_notification_rules` to be skipped - since the API for these notifications is disabled this test should not be necessary and won't run (unless I'm missing something?)